addons/sale_project/models/project.py

```diff
@@ -12,13 +12,12 @@
 class Project(models.Model):
     _inherit = 'project.project'
 
-    allow_billable = fields.Boolean("Billable", help="Invoice your time and material from tasks.")
+    allow_billable = fields.Boolean("Billable")
     sale_line_id = fields.Many2one(
         'sale.order.line', 'Sales Order Item', copy=False,
         compute="_compute_sale_line_id", store=True, readonly=False, index='btree_not_null',
         domain="[('is_service', '=', True), ('is_expense', '=', False), ('state', 'in', ['sale', 'done']), ('order_partner_id', '=?', partner_id), '|', ('company_id', '=', False), ('company_id', '=', company_id)]",
-        help="Sales order item to which the project is linked. Link the timesheet entry to the sales order item defined on the project. "
-        "Only applies on tasks without sale order item defined, and if the employee is not in the 'Employee/Sales Order Item Mapping' of the project.")
+        help="Sales order item that will be selected by default on the tasks and timesheets of this project, except if the employee set on the timesheets is explicitely linked to another sales order item on the project. It can be modified on each task and timesheet entry individually if necessary.")
```

Comment:
Sometimes you split the line in more when the sentence is too long and sometimes you put some `\n`. 🤔 
Why does the sentence here has no `\n`?